### ISSUE TITLE
Create signup page

### ISSUE THREAD

**Issue by terrameijar** [mpumzee/ChitChat-front#14]
Create sign up page.

Depends on #9

**Comment by im-machakata** [comment]
I'd be happy to work on this. You can assign me.

**Comment by terrameijar** [comment]
Perfect, thank you @im-machakata. Go for it. The first step is to clone the repository and set up a local development environment on your machine.

**Comment by terrameijar** [comment]
@im-machakata as discussed in the WhatsApp group, someone implemented the user sign-up feature at the last meetup(I apologise for that again). This issue still needs more work however, it would be nice to have unit tests to test the signup functionality, make some improvements to it and also to have it documented. Are you interested in working on any of these related tasks?

**Comment by im-machakata** [comment]
> @im-machakata as discussed in the WhatsApp group, someone implemented the user sign-up feature at the last meetup(I apologise for that again). This issue still needs more work however, it would be nice to have unit tests to test the signup functionality, make some improvements to it and also to have it documented. Are you interested in working on any of these related tasks?

I'm not good with unit tests. I'll see if I can document it, thanks.

**Comment by terrameijar** [comment]
> I'm not good with unit tests. I'll see if I can document it, thanks.

@im-machakata No problem. What do you think of the new sign up page? Were you able to check it out?

The best place to add Front End documentation is the [readme file in the `frontend` folder](https://github.com/PythonBulawayo/ChitChat/blob/main/frontend/README.md). We need to write the docs in such a way that anyone new to the project will know what pages and paths have been worked on. I suggest creating a section that lists the pages and the URL links to them using the following template as a guide:

# Project Name
Project Description

## Pages
- [Sign Up Page](link_to_sign_up_page)
- [Home Page](link_to_home_page)

**Comment by terrameijar** [comment]
Sign up and Sign in functionality in the backend is implemented and needs to be hooked up to the frontend
